src/nvim/ex_cmds.c

```diff
@@ -1635,14 +1635,14 @@ void print_line(linenr_T lnum, int use_number, int list)
 
   msg_start();
   silent_mode = FALSE;
-  info_message = TRUE;          /* use mch_msg(), not mch_errmsg() */
+  info_message = true;          // use mch_msg(), not mch_errmsg()
```

Comment:
general remark: we tend to reduce spaces to two spaces before `//` unless it is aligned with something else. (you don't need to change this everywhere, but it is good to keep in mind).